src/Util/BuildDirCopier.php

```diff
@@ -78,11 +82,12 @@ public function copy(string $dir, string $strategyName)
                         'Can\'t copy directory %s with strategy: %s',
                         $dir,
                         $strategyName
-                    )
+                    ),
+                    ['errorCode' => Error::WARN_COPY_MOUNTED_DIRS_FAILED]
```

Comment:
This message won't be written in the `cloud.error.log` file as `min_level` for error file handler is set to `warning`.
You can change `$this->logger->notice` to `$this->logger->warning` or decrese level to notice in `\Magento\MagentoCloud\Config\Log::getConfig`

Reply:
Changed to warning